addons/account/models/account_journal.py

```diff
@@ -643,7 +643,13 @@ def create_invoice_from_attachment(self, attachment_ids=[]):
                 invoice = self.env['account.move'].create({})
             invoice.with_context(no_new_invoice=True).message_post(attachment_ids=[attachment.id])
             invoices += invoice
+        return invoices
 
+    def create_invoice_from_attachment(self, attachment_ids=None):
+        '''
+         :return: A action redirecting to account.move tree/form view.
```

Comment:
The docstring is more important for this compared to `_generate_invoice_from_attachment` as it is a public method